### PR TITLE
Update set-up-phone-system-voicemail.md

### PR DESCRIPTION
**Pull request by carolynblanding**
@CarolynRowe cc @roykuntz 
Carolyn I discussed these changes with Roy. 

We need to add a reference back to the Section on configuring CVM with Exchange Server in this Doc https://docs.microsoft.com/en-us/skypeforbusiness/hybrid/plan-cloud-voicemail. Under the requirements for Exchange UM on-premises. We only refer to how to configure SfB, but not how to configure Exchange for CVM. Since we do here, we should cross reference back to here. 

cc: @dstrome

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by CarolynRowe** [comment]
There are typos in this PR.  Psystem should be Phone System....  @scanum can we get this fixed and then I'll merge the PR?  Thanks...

**Comment by scanum** [comment]
> There are typos in this PR. Psystem should be Phone System.... @scanum can we get this fixed and then I'll merge the PR? Thanks...

Hi @CarolynRowe I will have a freelancer check these typos.

File: Teams/set-up-phone-system-voicemail.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This article is for the [Office 365 admin](https://support.office.com/article/da
 > [!NOTE]
 > Cloud Voicemail supports depositing voicemail messages only in an Exchange mailbox and doesn't support any third-party email systems. 
 
-## Cloud-only environments: Set up Cloud Voicemail
+## Cloud-only environments: Set up Cloud Voicemail for Cloud Psystem Users
```
Comment (JohanFreelancer9) [inline Teams/set-up-phone-system-voicemail.md:33]:
```suggestion
## Cloud-only environments: Set up Cloud Voicemail for Cloud Phone System users
```

File: Teams/set-up-phone-system-voicemail.md [hunk 2]
```diff
@@ -40,22 +40,33 @@ For Skype for Business Online and Calling Plans users, Cloud Voicemail is automa
...
 
+## Set up Cloud Voicemail for Skype for Business Server Users
+
+The following information is about configuring Cloud Voicemail to work with users who are online for Exchange and on-premises for Skype for Business. 
+  
+1. You may need to purchase Exchange Online licenses to the people in your business. See [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md).
```
Comment (JohanFreelancer9) [inline Teams/set-up-phone-system-voicemail.md:61]:
```suggestion
1. You may need to purchase Exchange Online licenses for the users in your business. See [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md).
```

Comment (JohanFreelancer9) [inline Teams/set-up-phone-system-voicemail.md:61]:
```suggestion
1. You may need to purchase Exchange Online licenses for the people in your business. See [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md).
```

File: Teams/set-up-phone-system-voicemail.md [hunk 3]
```diff
@@ -40,22 +40,33 @@ For Skype for Business Online and Calling Plans users, Cloud Voicemail is automa
...
+    
+2. [Assign or remove licenses for Office 365 for business](https://support.office.com/article/997596b5-4173-4627-b915-36abac6786dc) the Exchange Online licenses to the people in your business.
+    
+3. Support for voicemail transcription has been added as of March 2017 and is enabled by default for all organizations and users. You can disable transcription for your organization by using Windows PowerShell and following the steps below.
+
+4. To configure Skype for Business server users for Cloud Voicemail please see [Plan Cloud Voicemail service for on-premises users](https://docs.microsoft.com/en-us/skypeforbusiness/hybrid/plan-cloud-voicemail)
```
Comment (JohanFreelancer9) [inline Teams/set-up-phone-system-voicemail.md:67]:
```suggestion
4. To configure Skype for Business server users for Cloud Voicemail, please see [Plan Cloud Voicemail service for on-premises users](https://docs.microsoft.com/en-us/skypeforbusiness/hybrid/plan-cloud-voicemail).
```

File: Teams/set-up-phone-system-voicemail.md [hunk 4]
```diff
@@ -40,22 +40,33 @@ For Skype for Business Online and Calling Plans users, Cloud Voicemail is automa
...
 
+## Set up Cloud Voicemail for Skype for Business Server Users
+
+The following information is about configuring Cloud Voicemail to work with users who are online for Exchange and on-premises for Skype for Business. 
+  
+1. You may need to purchase Exchange Online licenses to the people in your business. See [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md).
+    
+2. [Assign or remove licenses for Office 365 for business](https://support.office.com/article/997596b5-4173-4627-b915-36abac6786dc) the Exchange Online licenses to the people in your business.
```
Comment (JohanFreelancer9) [inline Teams/set-up-phone-system-voicemail.md:63]:
```suggestion
2. To assign the Exchange Online licenses to users in your organization, please see [Add users individually or in bulk](https://support.office.com/article/997596b5-4173-4627-b915-36abac6786dc).
```

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**